safehttp/plugins/xsrf/xsrf.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xsrf
+
+import (
+	"github.com/google/go-safeweb/safehttp"
+	"golang.org/x/net/xsrftoken"
+)
+
+const (
+	// TokenKey is the form key used when sending the token as part of POST
+	// request.
+	TokenKey = "xsrf-token"
+)
+
+// UserIDStorage stores the web application users' IDs,
+// needed in generating the XSRF token.
+type UserIDStorage interface {
+	GetUserID(*safehttp.IncomingRequest) (string, error)
+}
```

Comment:
`UserIDStorage` -> `UserIdentifier`. There doesn't have to be any storage, right?

`GetUserID` -> `UserID(*safehttp.IncomingRequest) (string, error)` -- in Go we avoid words like `Get` -- they don't add much value to the name